package-lock.json

```diff
@@ -3420,13 +3437,15 @@
           "version": "1.0.2",
           "resolved": "https://registry.npmjs.org/wrappy/-/wrappy-1.0.2.tgz",
           "integrity": "sha1-tSQ9jz7BqjXxNkYFvA0QNuMKtp8=",
-          "dev": true
+          "dev": true,
+          "optional": true
         },
         "yallist": {
           "version": "3.0.2",
           "resolved": "https://registry.npmjs.org/yallist/-/yallist-3.0.2.tgz",
           "integrity": "sha1-hFK0u36Dx8GI2AQcGoN8dz1ti7k=",
-          "dev": true
+          "dev": true,
```

Comment:
Did we want all of these, @HarshKhandeparkar @Harshithpabbati ? Just checking!

Reply:
@jywarren I think you reviewed a commit not the whole changes.

Reply:
Yes, i think the whole thing looks fine, but wasn't sure these package-lock.json changes were intended to be added? Thanks!